src/step/releases/mod.rs

```diff
@@ -325,7 +331,22 @@ pub(crate) fn release(run_type: RunType) -> Result<RunType, Error> {
                 package_to_release.package.assets.as_ref(),
                 &tag,
             )?;
-        } else {
+        }
+
+        if let Some(ref gitea_config) = gitea_config {
+            state.gitea = gitea::release(
+                package_to_release.package.name.as_ref(),
+                &package_to_release.release,
+                state.gitea,
+                gitea_config,
+                &mut dry_run_stdout,
+                package_to_release.package.assets.as_ref(),
+                &tag,
+            )?;
+        }
```

Comment:
Do you think I should set up a mirror of this repo on a Gitea instance and keep releases up to date there? Might help me catch issues if this integration ever gets out of date 🤔

Reply:
I do use knope myself, so if it gets outdated i'll likely make an issue (and if i have time i'll also take care of implementation. 

I did see that you invited me
![04F58095-396D-4818-B381-22D4B9063374](https://github.com/knope-dev/knope/assets/42149402/78183c63-787d-4094-a981-5805c43c7e77)

But testing the main repo is obviously always a good idea, so i support the idea ^^